targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52810/device/system_nrf52.c.bu

```diff
@@ -0,0 +1,365 @@
+/*
```

Comment:
Question to @0xc0170 and @marcuschangarm : Why is this file required?

Reply:
Sorry, that's not meant to be checked in